### PR TITLE
Qf 287 fat web front end changes feedback

### PR DESCRIPTION
**Pull request by anilkdhiman6**

### CODE REVIEW

File: src/SFA.DAS.FAT.Web/Views/Courses/CourseProviderDetail.cshtml [hunk 1]
```diff
@@ -4,9 +4,14 @@
 @using SFA.DAS.FAT.Web.Extensions
 @using DeliveryModeType = SFA.DAS.FAT.Web.Models.DeliveryModeType
 
+@section feedbackstyles {
+    <link rel="stylesheet" media="screen" href="https://www.gov.uk/assets/government-frontend/application-0e6f1b796b951ca0e07ce65f537ba3045e163b208c897f53b1fd4f36993385cc.css"/>
```
Comment (CodeWolf86) [inline src/SFA.DAS.FAT.Web/Views/Courses/CourseProviderDetail.cshtml:8]: You probably want to download these styles locally and reference them as part of the project. There isn't a guarantee that will stay live nor that they won't change. I can see that it's a front end assets site but I'm not familiar with it being used by AS, I'd ask James King, but aware he's on holiday.

File: src/SFA.DAS.FAT.Domain/Courses/Provider.cs [hunk 2]
```diff
@@ -69,8 +69,14 @@ public class FeedbackDetail
 
     public class FeedbackAttributes
     {
-        public List<string> Strengths { get; set; }
-        public List<string> Weaknesses { get; set; }
+        public List<FeedbackAttributeDetail> FeedbackAttributeDetail { get; set; }
```
Comment (CodeWolf86) [inline src/SFA.DAS.FAT.Domain/Courses/Provider.cs:72]: This may change if we simplify the API side to reduce nesting

**Review by CodeWolf86 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.FAT.Domain/Courses/Provider.cs (hunk 2 above)
Comment (anilkdhiman6) [reply src/SFA.DAS.FAT.Domain/Courses/Provider.cs]: Done

**Review by jamesmking — COMMENTED** [review]
This does need some frontend attention, lots of changes needed. Happy to take a look

**Review by CodeWolf86 — DISMISSED** [review]

**Review by CodeWolf86 — APPROVED** [review]

**Pull request merged**